src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalInternalUserFunctionalTest.java

```diff
@@ -1231,4 +1233,62 @@ private static List<Map<String, Object>> sortByValue(final List<Map<String, Obje
                 .sorted(Comparator.comparing(map -> (String) map.get(key)))
                 .collect(Collectors.toList());
     }
+
+    @Test
+    @ExtendWith(FeatureToggleConditionExtension.class)
+    void updateContactInformationForOrganisationShouldReturnSuccess() {
```

Comment:
add another 2 tests
1)  to update dx address details alone and assert updated data
2) to update address details without dx address details and assert updated data